src/main/java/lotto/vo/LottoGrade.java

```diff
@@ -0,0 +1,46 @@
+package lotto.vo;
+
+public enum LottoGrade {
+
+    MATCHES_ALL,
+    MATCHES_5,
+    MATCHES_4,
+    MATCHES_3,
+    MATCHES_NONE;
+
+    private static final int PRIZE_FOR_MATCHES_ALL = 2000000000;
```

Comment:
Java 7부터는 `100_000_000`처럼 숫자에 밑줄을 사용하여 가독성을 높일 수 있어요.

```suggestion
    private static final int PRIZE_FOR_MATCHES_ALL = 2_000_000_000;
```